response_operations_ui/views/reporting_units.py

```diff
@@ -169,19 +171,62 @@ def edit_contact_details(ru_ref, respondent_id):
     return redirect(url_for('reporting_unit_bp.view_reporting_unit', ru_ref=ru_ref))
 
 
-@reporting_unit_bp.route('/', methods=['GET', 'POST'])
+@reporting_unit_bp.route('/', methods=['GET'])
 @login_required
-def search_reporting_units():
-    form = SearchForm(request.form)
-    breadcrumbs = [{"text": "Reporting units"}]
-    business_list = None
+def search_reporting_unit_home():
+    return render_template('reporting-unit-search/reporting-units-search.html',
+                           form=RuSearchForm(),
+                           breadcrumbs=[{"text": "Reporting units"}])
+
+
+@reporting_unit_bp.route('/', methods=['POST'])
+@login_required
+def search_redirect():
+    form = RuSearchForm(request.form)
 
     if form.validate_on_submit():
         query = request.form.get('query')
 
-        business_list = reporting_units_controllers.search_reporting_units(query)
+        return redirect(url_for('reporting_unit_bp.search_reporting_units', query=query))
 
-    return render_template('reporting-units.html', business_list=business_list, form=form, breadcrumbs=breadcrumbs)
+
+@reporting_unit_bp.route('/search', methods=['GET'])
+@login_required
+def search_reporting_units():
+    search_key_words = request.values.get('query', '')
+    page = request.values.get('page', '1')
+    limit = app.config["PARTY_BUSINESS_RESULTS_PER_PAGE"]
```

Comment:
A future iteration of this should have this be a number sent to us by the client.  But totally fine for the first stab at it

Reply:
Good catch , I think your right . I put the setting at the point of entry so it would be easy to change , and then did the wrong thing anyway , Doh!  I think respondent search does something similar , but sets the limit further away from the route handler func in the controller.  Moving respondent to the view so that its trivial to pass them in if we want to add them later